mne_connectivity/spectral.py

```diff
@@ -662,15 +662,15 @@ def spectral_connectivity(data, names=None, method='coh', indices=None,
     as "brain connectivity" (i.e. values between two nodes indicate the
     connection strength). These are estimates in a statistical sense and thus
     are subject to noise in the estimation procedure. One can commonly convert
-    these to "brain connectivity" results by thresholding values below a certain
-    number. This should be done through some cross-validation procedure if
-    possible.
-
-    These measures are not to be intepreted over time. Each Epoch passed into
-    the dataset is intepreted as an independent sample of the same connectivity
-    structure. Within each Epoch, it is assumed that the spectral measure is
-    stationary. **Thus, spectral measures computed with only one Epoch will result
-    in errorful values.**
+    these to "brain connectivity" results by thresholding values below a
+    certain number. This should be done through some cross-validation procedure
+    if possible.
```

Comment:
Okay that's a good point, I can add some discussion there on what you wrote.

Regarding what I wrote originally: I would say that is a special case then of directed/non-directed measure of connectivity, where the thresholding step is analogous to an alpha value in hypothesis testing, which controls for false negatives. Thus would we agree then it is a notion of computing "connectivity", but just one specific model?

It sounds like (and I agree) that you would like me though to generalize the text here, which I will do.

Reply:
Yes, I would leave this out here and rather discuss in a tutorial about statistics. The way it is written now, I also am not sure how much information it would give a new user: questions that pop up are e.g. thresholding how, is not every "brain measure" a pure estimate (even when statistically controlled), etc.
I think we rather want to acknowledge that the opinions on "what can be called connectivity" might differ - and converge on one terminology that we keep throughout the toolbox as to not confuse users further.

Reply:
Okay sounds good. took it out.